src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,128 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private static StorageDaoImpl database;
+    private static RegistrationService registration;
+
+    @BeforeAll
+    static void beforeAll() {
+        database = new StorageDaoImpl();
+        registration = new RegistrationServiceImpl();
+    }
+
+    @Test
+        void register_userAgeIsLessThan18_NotOk() {
+        User user = new User();
```

Comment:
User user = new User(); in every test
maybe we can initialize user in BeforeEach and clean up in every test